src/scheduler/Analysis.py

```diff
@@ -137,10 +137,21 @@ def get_plugin_dict(self):
             mandatory_flag = plugin in MANDATORY_PLUGINS
             for key in default_plugins.keys():
                 default_flag_dict[key] = plugin in default_plugins[key]
-            result[plugin] = (self.analysis_plugins[plugin].DESCRIPTION, mandatory_flag, dict(default_flag_dict), self.analysis_plugins[plugin].VERSION)
+                result[plugin] = (self.analysis_plugins[plugin].DESCRIPTION, mandatory_flag, dict(default_flag_dict), self.analysis_plugins[plugin].VERSION, self.analysis_plugins[plugin].DEPENDENCIES)
         result['unpacker'] = ('Additional information provided by the unpacker', True, False)
         return result
 
+    def get_plugin_catalog(self):
```

Comment:
This function has to be added to Mock Objects in *src/test/integration/intercom* since it causes tests to fail otherwise:
- src/test/integration/intercom/test_task_communication.py
- src/test/integration/intercom/test_backend_scheduler.py
- src/test/integration/web_interface/rest/test_rest_binary.py